server/storage/sql_models.go

```diff
@@ -26,8 +60,11 @@ func CreateTUFTable(db gorm.DB) error {
 	}
 	query = db.Model(&TUFFile{}).AddUniqueIndex(
 		"idx_gun", "gun", "role", "version")
-	if query.Error != nil {
-		return query.Error
-	}
-	return nil
+	return query.Error
+}
+
+// CreateChangefeedTable creates the DB table for Changefeed
+func CreateChangefeedTable(db gorm.DB) error {
+	query := db.Set("gorm:table_options", "ENGINE=InnoDB DEFAULT CHARSET=utf8").CreateTable(&Changefeed{})
```

Comment:
It's possible we may want an index on gun as we'll filter by gun for single repo updates. No need for an index on version though as unless something is desperately wrong, it'll order the same way as the id and we shouldn't be querying the changefeed table for a specific version.